src/app/App.js

```diff
@@ -30,6 +31,34 @@ class App {
     this.button = new Button('button', playBtnText, 'play-button');
     this.rankingBox = new RankingBox('ranking-box', scores);
     this.modal = new Modal('modalBox');
+
+    const answers = [{
+      questionPicture: "../../static/assets/img/modes/people/1.jpg",
+      playerAnswer: "DarthVader",
+      computerAnswer: "Darth Father",
+      correctAnswer: "Luke Skywalker",
+      playerAnswerIsCorrect: false,
+      computerAnswerIsCorrect: false,
+    },
+    {
+      questionPicture: '../../static/assets/img/modes/people/36.jpg',
+      playerAnswer: "Jar Jar Binks",
+      computerAnswer: "Jar Jar Binks",
+      correctAnswer: "Jar Jar Binks",
+      playerAnswerIsCorrect: true,
+      computerAnswerIsCorrect: true,
+    },
+    ];
+    const closeWindow = (name, punctation) => {
+      console.log(name, punctation);
+      this.modal.closeModal();
+    };
+    this.gameOverScreen = new GameOverScreen(answers, closeWindow, 'modalBox');
+    
+    this.mainMenuPanel = new MainMenu('mainMenu', ['People', 'Vehicles', 'Starships'], 0);
+    this.whiteButton = new WhiteButton('whiteButton', 'Hall of fame', 'whiteButton');
+    this.whiteButton.addIcon('../../static/assets/ui/hof.svg');
+
     this.rules = new Rules('Mode Rules', 'rules');
     this.mainMenuPanel = new MainMenu(
```

Comment:
2x MainMenu i Hall of fame
![image](https://user-images.githubusercontent.com/33134609/104126810-0244e480-535f-11eb-888c-6013c77afbbf.png)
Wygląda na źle rozwiązane konflikty